src/main/resources/freemarker/es7x/index/log.ftl

```diff
@@ -13,7 +13,8 @@
     "date" : "${date}",
     "_id" : "${log.getRequestId()}",
   </#if>
-  "api":"${log.getApi()}"
+  "api":"${log.getApi()}",
+  "api-name":"${log.getApiName()}"
```

Comment:
can you check if it is necessary to add this to avoid errors if the name contains `"`?
```suggestion
  "api-name":"${log.getApiName()?j_string}"
```

to be checked on all freemarker modifications